codex/contracts/validation.nim

```diff
@@ -0,0 +1,3 @@
+import ./validation/validationparams
+
+export validationparams
```

Comment:
If you really think this is needed, it should probably be at the `/codex` level.

Reply:
I think it is not yet needed at this time, but while adding historical logging it may become handy. It it is still overkill after that I will just probably leave the `validation.nim` file at the contracts level. The reason I have put it on the `contracts` level is that `client` and `host` it is part of the node's contract (see https://github.com/codex-storage/nim-codex/blob/1e2ad956591d41ec93ede1dec7da228a4558e9d3/codex/codex.nim#L146):

```nim
s.codexNode.contracts = (client, host, validator)
```

Reply:
Ah ok, good point 👍  Yeah, I don't know how necessary this file is right now, but I'll leave it up to you to decide.

Reply:
I'm still thinking it might be best to keep everything "validation related" at the `/codex` level, or move everything to `/codex/contracts`. At the moment, prior to this PR, there is already a `validation` module at the `/codex`. I think introducing one in the `/codex/contracts` level will be confusing.

Reply:
I do not mind to move it to one level, but I would like to keep the `validationParams` (or from know most probably `validationConfig`) in a separate file (it will be then `validationconfig.nim`). The easiest would be to move that file to the top level next to the existing `validation.nim`. Would you agree?

Reply:
Yep that sounds good 👍